credsweeper/utils/util.py

```diff
@@ -30,7 +39,7 @@ def get_extension(cls, file_path: str) -> str:
         return extension
 
     @classmethod
-    def get_keyword_pattern(cls, keyword: str, separator: Separator = Separator.common) -> regex.Pattern:
+    def get_keyword_pattern(cls, keyword: str, separator: str = Separator.common) -> regex.Pattern:
```

Comment:
Hm, for some reason I thought that  `Separator` is a Enum, not just class with constants